caches/infinispan/src/main/java/org/commonjava/maven/galley/cache/infinispan/FastLocalCacheProvider.java

```diff
@@ -93,26 +100,82 @@
     protected FastLocalCacheProvider()
     {
         nfsBaseDir = System.getProperty( NFS_BASE_DIR_KEY );
```

Comment:
Once we have things closer to a final state, I'd like to replace this with a configuration object that we pass in. That gives us the option to enrich the config object without changing the API of the provider...and it allows us to avoid using static system properties.